scripts/get_xva_bridge.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# extract and read ova.xml
+info_ovaxml()
+{
+    TMPFOLD=$(mktemp -d /tmp/xvaXXXX)
+    tar -xvf $pathXVA/$nameXVA -C $TMPFOLD ova.xml
+    chmod +r $TMPFOLD/ova.xml
+    MyXmlValue=$(grep -oE "<member><name>bridge</name><value>[^<]*</value></member>" $TMPFOLD/ova.xml)
+    MyLenght=${#MyXmlValue}
+    MyLenghtMinus=$(($MyLenght-17))
+    MyNetworkValue=$(cut -c 1-$MyLenghtMinus <<< $MyXmlValue)
+    BridgeNetwork=$(cut -c 35-${#MyNetworkValue} <<< $MyNetworkValue)
+
+    if [ -d $TMPFOLD ]; then
+        rm -Rf $TMPFOLD
+    fi
+}
+
+if [ ! -z $1 ]; then
+    nameXVA=$(basename $1)
+    pathXVA=$(dirname $1)
+    if [ ! -e $1 ]; then
+        echo "File $1 doesn't exist. check the name ou the path."
+        exit 0
+    fi
+else
+    echo "Error: you forget to give the file to scan"
+    exit 0
+fi
+
+info_ovaxml
+echo "$nameXVA bridge network is: $BridgeNetwork and his compression's method is: $(file $pathXVA/$nameXVA | cut -f 2 -d :  | cut -f 2 -d " ")."
```

Comment:
```suggestion
echo "${nameXVA}'s bridge network is: $BridgeNetwork and its compression method is: $(file $pathXVA/$nameXVA | cut -f 2 -d :  | cut -f 2 -d " ")."
```

I fear the method used to get the compression method may be fragile (depends on `file`'s output)

Reply:
Anyway I don't have a better way to suggest, at least for such a script.